cpp/cmake/thirdparty/get_cufile.cmake

```diff
@@ -16,7 +16,7 @@
 function(find_and_configure_cufile)
 
   list(APPEND CMAKE_MODULE_PATH ${CUDF_SOURCE_DIR}/cmake/Modules)
-  rapids_find_package(cuFile QUIET)
+  rapids_find_package(cuFile)
```

Comment:
Note: I made this change because it's helpful for debugging issues. It's not necessary for CUDA 12 but I do think it's better to leave this way.